driver/SDL/modSDL.c

```diff
@@ -1,6 +1,14 @@
-#include "../include/common.h"
+[B#include "../include/common.h"
```

Comment:
`[B` typo before `#include`?

Reply:
Sorry! I didn't compile-test it before pushing.